src/schema/printer.ts

```diff
@@ -63,6 +66,8 @@ export class SchemaPropertyRenderer {
 /**
  * Prints a JSON schema in a human-friendly format.
  *
+ * @deprecated
```

Comment:
to my knowledge, this is not used.  So I deprecated it and deleted the test.  Check me on that.